README.md

```diff
@@ -194,6 +195,34 @@ This also works with rules that are applied to the element via a class name for
 which some rules are defined in a stylesheet currently active in the document.
 The usual rules of css precedence apply.
 
+### `toBeVisible`
+
+This allows you to check if an element is currently visible to the user.
+
+An element is visible if **all** the following conditions are met:
+
+* it does not have its css property `display` set to `none`
+* it does not have its css property `visibility` set to either `hidden` or
+  `collapse`
+* its parent element is also visible (and so on up to the top of the DOM tree)
```

Comment:
What about `opacity: 0`? That's a pretty common way to make things invisible as well. Hmmm...

Reply:
Hmmmmm, yes. Good point 🤔